pkg/codegen/tmpl/gen_header.tmpl

```diff
@@ -1,4 +1,4 @@
-// THIS FILE IS GENERATED. EDITING IS FUTILE.
+// Code generated. DO NOT EDIT.
```

Comment:
This doesn't actually match the regexp - there needs to be a space after `generated`.

```suggestion
// Code generated - EDITING IS FUTILE. DO NOT EDIT.
```

Reply:
Good catch, thanks for checking thoroughly!

I'll make the proposed changes, and regen.